src/util/normalize-manifest/fix.js

```diff
@@ -308,7 +308,7 @@ export default (async function(
     }
   }
 
-  for (const dependencyType of DEPENDENCY_TYPES) {
+  for (const dependencyType of ['resolutions', ...DEPENDENCY_TYPES]) {
```

Comment:
what do you think about adding `resolutions` in the constants file 

and doing something like 
```
RESOLUTIONS = 'resolutions';
MANIFEST_FIELDS = [RESOLUTIONS, ...DEPENDENCY_TYPES];
```
then doing
```
for (const fields of MANIFEST_FIELDS) {
```
since resolution is not really a dependencyType? And we can add more fields to add comments to later if we want to?

Reply:
Good call. I pushed up this change 👍